Kickstarter-iOS/Views/DiscoveryProjectCategoryView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  DiscoveryProjectCategoryView.swift
+//  Kickstarter-iOS
+//
+//  Created by Isabel Barrera on 4/30/18.
+//  Copyright © 2018 Kickstarter. All rights reserved.
+//
+
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+@IBDesignable internal final class DiscoveryProjectCategoryView: UIView, NibLoading {
+  private let viewModel: DiscoveryProjectCategoryViewModelType = DiscoveryProjectCategoryViewModel()
+
+  @IBOutlet private weak var blurView: UIImageView!
+  @IBOutlet private weak var categoryStackView: UIStackView!
+  @IBOutlet private weak var categoryViewImageView: UIImageView!
+  @IBOutlet private weak var categoryViewLabel: UILabel!
+
+  public required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  internal func configureWith(name: String, imageNameString: String) {
```

Comment:
Here we could use only one input and pass the name and imageName, like: `viewModel.inputs.configureWith(name: name, imageNameString: imageNameString)`.